src/components/MovieMain/MovieMain.js

```diff
@@ -3,12 +3,13 @@ import React, { Component } from 'react'
 import MovieTile from '../MovieTile/MovieTile';
 
 class MovieMain extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
       loading: 'All Movies Trying To Load...',
-      movies: []
+      movies: [],
+      error: ''
```

Comment:
I like this implimentation of error messages being held on state to insure things updated when they should, and allows dom manipulation to reflect what I wanted alerts to say.